src/inverseRobot.js

```diff
@@ -8,6 +8,36 @@
 
 function inverseRobot(robot) {
   // write code here
+  const ONE_VALUE_IN_OBJECT = 1;
+  let countsObj = {};
+
+  const resObj = {};
+
+  if (Object.keys(robot).length === 0) {
+    return {};
+  }
+
+  for (const key in robot) {
```

Comment:
It's better to use 'for (const [key, value] of Object.entries(robot))' for iterating through the object